scenarios/CreateAKSDeployment/README.md

```diff
@@ -20,17 +20,11 @@ The first step in this tutorial is to define environment variables.
 
 ```bash
 export RANDOM_ID="$(openssl rand -hex 3)"
-export NETWORK_PREFIX="$(($RANDOM % 254 + 1))"
 export SSL_EMAIL_ADDRESS="$(az account show --query user.name --output tsv)"
-export MY_RESOURCE_GROUP_NAME="myAKSResourceGroup$RANDOM_ID"
+export MY_RESOURCE_GROUP_NAME="myResourceGroup"
```

Comment:
Resource group name should remain as myAKSResourceGroup$RANDOM_ID.  It being "myResourceGroup" will cause repeated executions to always fail